src/codegen/llvm.zig

```diff
@@ -4188,41 +4188,166 @@ pub const FuncGen = struct {
         return self.builder.buildInsertValue(partial, len, 1, "");
     }
 
+    inline fn isPowerOfTwo(bits: u64) bool {
+        return bits != 0 and ((bits & (~bits + 1)) == bits);
+    }
+
+    fn intTypeFromBitsAndSignRounded(self: *FuncGen, bits: u16, signed: bool) error{OutOfMemory}!Type {
```

Comment:
I think this function is awkward and unnecessary. Allow me to suggest an alternate implementation at the callsites...